popcorndao/workspace
# Feature/Validate Ethereum address when adding contract to emissions dashboard

## Description

This PR adds contract address validation when adding a contract to the dashboard. It implements tickets [#348](https://app.zenhub.com/workspaces/popcorn-603eb49d1a9a0b0011b49b36/issues/popcorndao/workspace/348) and #349 

## To Test
- Check out this branch
- Cd into the emission-dashboard directory and run `yarn dev`
- Run the contracts service
- Using an address of one of the ethereum addresses, add that address to the dashboard.
